### DISCUSSION TITLE
The search engine should ignore accents

### DISCUSSION THREAD

**Discussion by Orybon** [mealie-recipes/mealie#2123] [Feature Request]
**Before submitting this feature request I have**

- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
The search engine is accent sensitive. While importing recipes on mealie the accent are gathered from original url. Often users (we are all lazy :) ) search string without accent and so the search engine do not find any occurence.

**(Optional): Suggest A Solution**
Problably alter SQL requests ? 
Maybe add an option in the settings to enable this ?

**Comment by fleshgolem** [comment]
This should work correctly on nightly now

**Reply by Orybon** [reply]
Thank you very much, just pulled last nightly and it works like a charm.
